metric_tank/cassandra.go

```diff
@@ -71,20 +71,32 @@ func InitCassandra() error {
 	cluster.NumConns = *cassandraWriteConcurrency
 	cSession, err = cluster.CreateSession()
 
-	CassandraWriteQueue = make(chan *ChunkWriteRequest, *cassandraWriteQueueSize)
+	CassandraWriteQueue = make(chan *ChunkWriteRequest)
```

Comment:
why do we need CassandraWriteQueue.  Can't we just create a function for adding items to the queue which automatically selects the correct workQueue to feed a message to.
eg. 
```
func QueueWrite(c *ChunkWriteRequest) {
   //https://github.com/OneOfOne/xxhash
   workQueues[xxhash.Checksum32([]byte(c.key))%len(workQueues)] <- c
}
```